### PR TITLE
Update the Fabric8 Kubernetes client to 4.13

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Task

### Description

This Pr updates the Fabric8 Kubernetes Client to a new version. Once updated, we should be able to fix the #3779 workaround.

### Checklist

- [x] Make sure all tests pass
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally

### CODE REVIEW

File: api/src/test/resources/io/strimzi/api/kafka/model/040-Crd-kafka-crdApi-v1.yaml [hunk 1]
```diff
@@ -1589,6 +1596,13 @@ spec:
                                     type: string
                                   user:
                                     type: string
+                              seccompProfile:
```
Comment (ppatierno) [inline api/src/test/resources/io/strimzi/api/kafka/model/040-Crd-kafka-crdApi-v1.yaml:1599]: from where this `seccompProfile` comes? what is that?

Comment (scholzj) [reply api/src/test/resources/io/strimzi/api/kafka/model/040-Crd-kafka-crdApi-v1.yaml:1599]: New versions of Fabric8 often adopt newer Kubernetes schemas which have new fields. And that can change the CRDs in places where we use the Kubernetes APIs directly - here in this case the security context.

**Review by ppatierno — APPROVED** [review]

**Pull request merged**